### PR TITLE
Prep 3.3.0 release

### PR DESCRIPTION
**Pull request by alinabuzachis**
##### SUMMARY

Run add_docs, generate changelog

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

several

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -5,6 +5,37 @@ community.aws Release Notes
...
+======
+
+Minor Changes
+-------------
+
+- aws_ec2 inventory - Allow for literal strings in hostname that don't match filter parameters in ec2 describe-instances (https://github.com/ansible-collections/amazon.aws/pull/826).
+- aws_ssm - Add support for ``endpoint`` parameter (https://github.com/ansible-collections/amazon.aws/pull/837).
+- module.utils.rds - add retry_codes to get_rds_method_attribute return data to use in call_method and add unit tests (https://github.com/ansible-collections/amazon.aws/pull/776).
+- module.utils.rds - refactor to utilize get_rds_method_attribute return data (https://github.com/ansible-collections/amazon.aws/pull/776).
+- module_utils - add new aliases ``aws_session_token`` and ``session_token`` to the ``security_token`` paramater to be more in-line with the boto SDK (https://github.com/ansible-collections/amazon.aws/pull/631).
```
Comment (jatorcasso) [inline CHANGELOG.rst:18]:
```suggestion
- module_utils - add new aliases ``aws_session_token`` and ``session_token`` to the ``security_token`` parameter to be more in-line with the boto SDK (https://github.com/ansible-collections/amazon.aws/pull/631).
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -5,6 +5,37 @@ community.aws Release Notes
...
+- module_utils.rds - Add support and unit tests for addition/removal of IAM roles to/from a db instance in module_utils.rds with waiters (https://github.com/ansible-collections/amazon.aws/pull/714).
+
+Bugfixes
+--------
+
+- Include ``PSF-license.txt`` file for ``plugins/module_utils/_version.py``.
+- aws_account_attribute lookup plugin - fix linting errors in documentation data (https://github.com/ansible-collections/amazon.aws/pull/701).
+- aws_account_attribute lookup plugin - fix linting errors in documentation data (https://github.com/ansible-collections/amazon.aws/pull/701).
```
Comment (jatorcasso) [inline CHANGELOG.rst:26]:
```suggestion

```

File: changelogs/changelog.yaml [hunk 3]
```diff
@@ -743,3 +743,51 @@ releases:
...
     - 719-ec2_instance-fix-deprecation-warning-scope.yml
     release_date: '2022-04-05'
+  3.3.0:
+    changes:
+      bugfixes:
+      - Include ``PSF-license.txt`` file for ``plugins/module_utils/_version.py``.
+      - aws_account_attribute lookup plugin - fix linting errors in documentation
+        data (https://github.com/ansible-collections/amazon.aws/pull/701).
+      - aws_account_attribute lookup plugin - fix linting errors in documentation
+        data (https://github.com/ansible-collections/amazon.aws/pull/701).
```
Comment (jatorcasso) [inline changelogs/changelog.yaml:753]:
```suggestion

```

File: changelogs/changelog.yaml [hunk 4]
```diff
@@ -743,3 +743,51 @@ releases:
...
+      minor_changes:
+      - aws_ec2 inventory - Allow for literal strings in hostname that don't match
+        filter parameters in ec2 describe-instances (https://github.com/ansible-collections/amazon.aws/pull/826).
+      - aws_ssm - Add support for ``endpoint`` parameter (https://github.com/ansible-collections/amazon.aws/pull/837).
+      - module.utils.rds - add retry_codes to get_rds_method_attribute return data
+        to use in call_method and add unit tests (https://github.com/ansible-collections/amazon.aws/pull/776).
+      - module.utils.rds - refactor to utilize get_rds_method_attribute return data
+        (https://github.com/ansible-collections/amazon.aws/pull/776).
+      - module_utils - add new aliases ``aws_session_token`` and ``session_token``
+        to the ``security_token`` paramater to be more in-line with the boto SDK (https://github.com/ansible-collections/amazon.aws/pull/631).
```
Comment (jatorcasso) [inline changelogs/changelog.yaml:780]:
```suggestion
        to the ``security_token`` parameter to be more in-line with the boto SDK (https://github.com/ansible-collections/amazon.aws/pull/631).
```

**Review by jatorcasso — CHANGES_REQUESTED** [review]
tiny typo and duplicate line changes

**Comment by alinabuzachis** [comment]
> tiny typo and duplicate line changes

@jatorcasso Thank you!

**Review by jatorcasso — APPROVED** [review]
LGTM!

**Review by markuman — APPROVED** [review]

**Review by tremble — APPROVED** [review]

**Pull request merged**